### PR TITLE
refs #54624 admin show edit delete create user

### PR DESCRIPTION
**Pull request by NeverSmileK57CLC**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class Admin::UsersController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @users = User.all
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: read again about cancancan

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    redirect_to admin_users_path
+  end
+
+  private
+  def user_params
+    params.require(:user).permit :email, :name, :chatwork_id, :password,
+      :password_confirmation
+  end
+
+  def user_update_params
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: why we need 2 type of params

File: app/views/admin/users/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<h1><%= t ".edit" %></h1>
+
+<%= form_for [:admin, @user] do |f| %>
+  <%= render "form_edit", f: f %>
```
Comment (hellovietnam93) [inline app/views/admin/users/edit.html.erb]: with one form render, i still do like like that.

File: app/views/admin/users/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+    </div>
+
+    <div class="form-group">
+      <%= f.label :chatwork_id %>
+      <%= f.text_field :chatwork_id, class: "form-control" %>
+    </div>
+
+    <% if params[:action] == "new" %>
```
Comment (hellovietnam93) [inline app/views/admin/users/_form.html.erb]: <% if f.object.new_record? %>

**Comment by hellovietnam93** [comment]
OK

File: app/helpers/sessions_helper.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+module SessionsHelper
+  def current_user? user
```
Comment (dieunb) [inline app/helpers/sessions_helper.rb]: Viết vào model nhé

File: config/routes.rb [hunk 6]
```diff
@@ -3,4 +3,7 @@
   devise_for :users
 
   resources :users, only: [:show]
+  namespace :admin do
```
Comment (dieunb) [inline config/routes.rb]: root page

**Pull request merged**